translations/README_de_DE.md

```diff
@@ -0,0 +1,86 @@
+<h1 align="center">
+  <br>
+  <a href="http://atlasos.net"><img src="https://i.imgur.com/xV08gIt.png" alt="Atlas" width="900"></a>
+</h1>
+<h4 align="center">Ein offenes und transparentes Windows-Betriebssystem, das zur Optimierung von Leistung und Latenzzeit entwickelt wurde.</h4>
+
+<p align="center">
+  <a href="https://github.com/Atlas-OS/Atlas/wiki/2.-Installing">Installation</a>
+  •
+  <a href="https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#contents">FAQ</a>
+  •
+  <a href="https://discord.com/servers/atlas-795710270000332800" target="_blank">Discord</a>
+  •
+  <a href="https://forum.atlasos.net/">Forum</a>
+</p>
+
+
+# Was ist Atlas?
+
+Atlas ist eine modifizierte Version von Windows, welche alle negativen Nachteile von Windows beseitigt, die sich negativ auf die Spieleleistung auswirken. Wir sind ein transparentes und quelloffenes Projekt, dass die Gleichberechtigung der Spieler anstrebt, unabhängig davon, ob Sie einen Kartoffel- oder einen Gaming-PC verwenden.
+
+Obwohl unser Hauptaugenmerk auf der Leistung liegt, sind wir auch eine hervorragende Option zur Reduzierung von System- und Netzwerklatenz, Eingabeverzögerung und zum Schutz Ihrer Privatsphäre.
+
+## Inhaltsübersicht
+
+- [FAQ](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ)
+  - [Was ist das Atlas-Projekt?](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#11-what-is-the-atlas-project)
+  - [Wie kann ich Atlas installieren?](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#12-how-do-i-install-atlas-os)
+  - [Was wird in Atlas entfernt?](https://github.com/Atlas-OS/Atlas/wiki/1.-FAQ#13-whats-removed-in-atlas-os)
+- <a href="#windows-vs-atlas">Windows vs. Atlas</a>
+- [Post Install](https://github.com/Atlas-OS/Atlas/wiki/3.-Post-Install)
+- [Branding-Kit](https://github.com/Atlas-OS/Atlas/blob/main/img/brand-kit.zip?raw=true)
```

Comment:
there is a "-" in Branding kit.
"kit" should be uncaptialized.

Reply:
I can do that but in german you write "Branding-Kit" or I could change it to "Marken-Kit" but it is more common to write "Branding-Kit" in german.

Reply:
in that case, just keep it as Branding-Kit.